cmd/destroy/destroy.go

```diff
@@ -95,6 +97,20 @@ func Destroy(ctx context.Context) *cobra.Command {
 		Args:  utils.NoArgsAccepted("https://okteto.com/docs/reference/cli/#destroy"),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if options.ManifestPath != "" {
```

Comment:
Is this code the same as in deploy? Could we extract a function common to both and test it with an unit test?

Reply:
yes, that is the same to up deploy and destroy. i'm working on it.